actors/evm/src/interpreter/precompiles.rs

```diff
@@ -33,6 +33,19 @@ pub fn assert_zero_bytes<const S: usize>(src: &[u8]) -> Result<(), PrecompileErr
     }
 }
 
+#[repr(u32)]
+pub enum BuiltinType {
+    NonExistent = 0,
+    // user actors are flattened to "system"
+    /// System includes any singletons not otherwise defined.
+    System = 1,
+    Embryo = 2,
+    Account = 3,
+    StorageProvider = 4,
+    EVMActor = 5,
+    OtherBuiltin = 6,
```

Comment:
This is "native", not "builtin". Well, for _now_ it's only builtin actors. But later this will be the "catchall".